Sources/StreamChatUI/ChatChannelList/SwipeableView_Tests.swift

```diff
@@ -0,0 +1,69 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import StreamChatTestTools
+@testable import StreamChatUI
+import XCTest
+
+class SwipeableView_Tests: XCTestCase {
+    func test_defaultAppearance() {
+        // Create SwipeableView
+        let view = SwipeableView().withoutAutoresizingMaskConstraints
+
+        // Simulate the delete and more buttons default behavior:
+        let (deleteView, moreView) = defaultActionViews(moreAction: {}, deleteAction: {})
+        view.actionItemsStackView.addArrangedSubview(deleteView)
+        view.actionItemsStackView.addArrangedSubview(moreView)
+        view.addSizeConstraints()
+        
+        // Simulate view moving to superview to have view initialized.
+        view.executeLifecycleMethods()
+        
+        // Buttons are revealed after swipe gesture so we need to simulate it to check the buttons.
+        view.swipeOpen()
+        
+        AssertSnapshot(view, variants: .onlyUserInterfaceStyles)
+    }
+
+    private func defaultActionViews(
+        moreAction: @escaping (() -> Void),
+        deleteAction: @escaping (() -> Void)
+    ) -> (UIView, UIView) {
+        let deleteButton = UIButton().withoutAutoresizingMaskConstraints
+
+        deleteButton.setImage(UIConfig.default.images.messageActionDelete, for: .normal)
+
+        deleteButton.backgroundColor = UIConfig.default.colorPalette.alert
+        deleteButton.tintColor = .white
+
+        let moreActionButton = UIButton().withoutAutoresizingMaskConstraints
+
+        moreActionButton.setImage(UIConfig.default.images.more, for: .normal)
+
+        moreActionButton.backgroundColor = UIConfig.default.colorPalette.background1
+        moreActionButton.tintColor = UIConfig.default.colorPalette.text
+
+        let deleteView = CellActionView().withoutAutoresizingMaskConstraints
+        deleteView.actionButton = deleteButton
+
+        let moreActionsView = CellActionView().withoutAutoresizingMaskConstraints
+        moreActionsView.actionButton = moreActionButton
+
+        return (moreActionsView, deleteView)
+    }
```

Comment:
IMO calling them `defaultActionViews` is not the best. It should be just some dummy test views. You don't want this test to fail when we tweak the "more" icon. I'd suggest creating just some 2 plain views with different background colors and that's it.

Reply:
Changed.